api/mlflow/mlflow.go

```diff
@@ -61,6 +61,7 @@ func (mlflow *client) doCall(req *request, resp interface{}) error {
 		return err
 	}
 
+	httpReq.Header.Set("Content-Type", "application/json")
```

Comment:
This is now required to be explicitly specified when sending requests to the Mlflow server https://mlflow.org/docs/latest/deployment/deploy-model-locally.html#accepted-input-formats.